misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -153,6 +154,19 @@ class JettyService @Inject internal constructor(
       server.addConnector(httpsConnector)
     }
 
+    if (webConfig.unix_domain_socket != null) {
+      val udsConnector = UnixSocketConnector(
+        server,
+        null /* executor */,
+        null /* scheduler */,
+        null /* buffer pool */,
+        webConfig.selectors ?: -1,
+        listOf(HttpConnectionFactory(httpConfig)).toTypedArray()
+      )
+      udsConnector.setUnixSocket(webConfig.unix_domain_socket.path)
+      server.addConnector(udsConnector)
```

Comment:
Right before this, you'll also want to register metrics. Here's how it's done with the other connectors:
```
      httpsConnector.addBean(connectionMetricsCollector.newConnectionListener(
          "https",
          webConfig.ssl.port
      ))
```

Reply:
Yes. @janzantinge please follow-up!